esmvaltool/cmorizers/data/formatters/datasets/esacci_landcover.py

```diff
@@ -0,0 +1,159 @@
+"""ESMValTool CMORizer for ESACCI-LANDCOVER pft data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    ftp://anon-ftp.ceda.ac.uk/neodc/esacci/land_cover/data/pft/
+
+Last access
+    20240626
+
+Download and processing instructions
+    Download the data from:
+      pft/v2.0.8/
+    Put all files under a single directory (no subdirectories with years).
+    in ${RAWOBS}/Tier2/ESACCI-LANDCOVER
+
+"""
+
+import os
+import glob
+import logging
+from datetime import datetime
+import iris
+import numpy as np
+
+from ...utilities import (
+    fix_coords,
+    fix_var_metadata,
+    set_global_atts,
+    add_typebare,
+    save_variable,
+)
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Enable the new split-attributes handling mode
+iris.FUTURE.save_split_attrs = True
+
+
+def average_block(data, block_size):
+    """Average the data within each block of size block_size."""
```

Comment:
could you maybe pls provide info on `block_size`? A numpy-style doc would be nice, but as long as the arguments are defined, I'm happy :)